src/main/java/edu/iis/powp/command/ComplexCommand.java

```diff
@@ -1,20 +1,43 @@
 package edu.iis.powp.command;
 
+import java.util.List;
 import java.util.Vector;
 
 import edu.iis.client.plottermagic.IPlotter;
 
 public class ComplexCommand implements PlotterCommand {
 	
-    Vector<PlotterCommand> plotterCommands = new Vector<PlotterCommand>();
-    public void add(PlotterCommand command) {
-       plotterCommands.add(command);
-   }
-    
+//    Vector<PlotterCommand> plotterCommands = new Vector<PlotterCommand>();
+//    public void add(PlotterCommand command) {
+//       plotterCommands.add(command);
+//   }
+//    
     
-    @Override public void execute(IPlotter plotter) {
+    private List<PlotterCommand> plotterCommands;
+	private CommandDrawLineToPosition commandDrawLineToPosition;
+	private CommandSetPosition commandSetPosition;
+
+	public ComplexCommand(List<PlotterCommand> plotterCommands) {
+    	this.plotterCommands = plotterCommands;
+	}
+
+	public ComplexCommand() {

```

Comment:
PLEASE FIX: The constructor should initialize plotterCommands attribute or there will be NullPointerExceptions!